packages/components/src/button/index.js

```diff
@@ -104,12 +106,18 @@ export function Button( props, ref ) {
 				// the tooltip is not explicitly disabled.
 				false !== showTooltip ) );
 
+	const descriptionId = describedBy ? uniqueId() : null;
```

Comment:
This `uniqueId` will change on every render, which is unnecessary.

I think it should use the `useInstanceId` hook.